pkg/storage/gcs/gcs.go

```diff
@@ -212,6 +212,13 @@ func (d *driver) StorageChanged(cr *imageregistryv1.Config) bool {
 }
 
 func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
+	// Previous status condition was set to failed because, of tag operation failure,
+	// means bucket creation was successful and hence will only try adding tags.
+	if cond := util.FetchCondition(cr, defaults.StorageTagged); cond.Status == operatorapi.ConditionFalse &&
+		cond.Reason == gcpTagsFailedStatusReason {
```

Comment:
let's change this condition around so that we check for anything other than success, instead of checking for the specific error? i.e
```
if cond := util.FetchCondition(cr, defaults.StorageTagged); cond.Status != operatorapi.ConditionTrue {
```

Reply:
suggestion incorporated. Condition updated and moved to end of the function.